packages/react-relay/relay-hooks/__tests__/useBlockingPaginationFragment-test.js

```diff
@@ -28,10 +28,9 @@ const {
   graphql,
 } = require('relay-runtime');
 const {createMockEnvironment} = require('relay-test-utils');
-const {
-  disallowWarnings,
-  expectWarningWillFire,
-} = require('relay-test-utils-internal');
+const {disallowWarnings, expectWarningWillFire} = (jest.requireActual(
+  'relay-test-utils-internal',
+): $FlowFixMe);
```

Comment:
Avoid module getting reset when `jest.resetModules` is called somewhere in this test, otherwise `expectWarningWillFire` errors.